src/NServiceBus.Core/ObjectBuilder/LightInject/LightInject.g.cs

```diff
@@ -21,12 +21,12 @@
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
     SOFTWARE.
 ******************************************************************************
-    LightInject version 6.2.1
+    LightInject version 6.3.5
     http://www.lightinject.net/
     http://twitter.com/bernhardrichter
 ******************************************************************************/
 
-//NOTE: This is LightInject 6.3.4: https://github.com/seesharper/LightInject/blob/v6.3.4/src/LightInject/LightInject.cs with all public types made internal.
+//NOTE: This is LightInject 6.4.0: https://github.com/seesharper/LightInject/blob/v6.4.0/src/LightInject/LightInject.cs with all public types made internal, and NET472 and NETCOREAPP2_0 added to the conditional compilation conditions..
```

Comment:
This note isn't correct, because you actually added `NETCOREAPP2_1` to the original file.

Reply:
👍 good catch. I amended the commit.